src/show_dialog.cpp

```diff
@@ -201,7 +201,7 @@ dialog_frame::dimension_measurements dialog_frame::layout(int x, int y, int w, i
 	}
 
 	y -= dim_.title.h;
-	w = std::max<int>(w,std::max<int>(int(dim_.title.w),int(buttons_width)));
+	w = std::max<int>(w,std::max<int>(dim_.title.w,int(buttons_width)));
```

Comment:
BTW the target type is Uint32, not int, if we're being pedantic. C++ will figure it out and do the right thing.